nats-tools/natscli.md

```diff
@@ -0,0 +1,57 @@
+# natscli
+
+A command line utility to interact with and manage NATS.
+
+This utility replaces various past tools that were named in the form `nats-sub` and `nats-pub`, adds several new capabilities and support full JetStream management.
+
+Check out the repo for more details: [github.com/nats-io/natscli](https://github.com/nats-io/natscli).
+
+## Installing `nats`
+
+For macOS:
+
+```
+> brew tap nats-io/nats-tools
+> brew install nats-io/nats-tools/nats
+```
+
+For Arch Linux:
+
+```
+> yay natscli
+```
+
+For Docker:
+
+```
+docker pull synadia/nats-server:nightly
```

Comment:
It’s in nats-box too. We should probably promote that instead

Reply:
Cool, I replaced `synadia/nats-server:nightly` with `synadia/nats-box:latest`.